packages/app-check/src/index.ts

```diff
@@ -55,7 +56,7 @@ function registerAppCheck(firebase: _FirebaseNamespace): void {
     )
   );
 
-  // TODO: register AppCheck version with firebase.registerVersion() before BETA. We don't want to report version in EAP
+  firebase.registerVersion(name, version);
```

Comment:
importing json file changes output folder structure of typings file. can you please double check that the typing field and other fields are pointing to valid files?

Reply:
The directory structure didn't change because `client.ts` was already importing package.json but the typings path was wrong anyway (maybe became wrong when `client.ts` originally was modified to import package.json) so I fixed it.